pkg/datastore/datastore.go

```diff
@@ -21,31 +21,43 @@ package datastore
 import (
 	"context"
 	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/vmware-labs/multi-tenant-persistence-for-saas/pkg/authorizer"
+	"github.com/vmware-labs/multi-tenant-persistence-for-saas/pkg/dbrole"
+)
+
+const (
+	// Constants for LOG field names & values.
+	LOG_LEVEL_ENV_VAR = "LOG_LEVEL"
```

Comment:
Since LOG_LEVEL is not logger's field's name or value but a name of env. variable, I propose  this line is moved to a separate `const( .... )` block